flake8_graphql.py

```diff
@@ -17,9 +17,23 @@ class GraphQLChecker(object):
     def __init__(self, tree, *args, **kwargs):
         self.tree = tree
 
+    @classmethod
+    def add_options(cls, parser):
+        parser.add_option(
+            '--gql-identifier',
+            type='string',
```

Comment:
I mean type=str. Types in argparse are either an identifier or a callable that returns the thing, and I think the latter is more generally applicable and nicer.